econml/tests/test_ortho_learner.py

```diff
@@ -0,0 +1,225 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+from econml._ortho_learner import _OrthoLearner, _crossfit
+from sklearn.linear_model import LinearRegression
+from sklearn.preprocessing import PolynomialFeatures
+from sklearn.linear_model import LinearRegression, LassoCV, Lasso
+from sklearn.model_selection import KFold
+import numpy as np
+import unittest
+import joblib
+
+
+class TestOrthoLearner(unittest.TestCase):
+
+    def test_crossfit(self):

```

Comment:
We need more tests surrounding the behavior of `_crossfit` when keyword and non-keyword arguments are None, to verify that the behavior matches the spec.

Reply:
Added some more. Let me know if enough. Please propose any other concrete cases.